RELEASE.md

```diff
@@ -0,0 +1,10 @@
+# Release Process
+
+This document describes the process for a public drand release.
+
+## Major / Minor release Process
+
+* A release is initiated by creating a protected release branch 'release-va.b' marking the minor release.
+* Once this branch is created, protected, and pushed, a signed tag should be created to indicate the first release candidate candidate 'va.b.0-rc1'
+* This tag will be turned by the `go-releaser` CI action into a release candidate on github, and will notify anyone who is watching releases of the new action.
```

Comment:
Can we have a step before this to create release notes, which enumerate the breaking changes (at minimum) but also new features and bugfixes? These notes need to be pasted into the release candidate created here.